functions/v2/ocr/app/test/index.test.js

```diff
@@ -0,0 +1,170 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+require('..');
```

Comment:
Is this needed? We should only require specifics.

Reply:
We need something along these lines just to evaluate the function calls. Previously we had `const program = require('..')` and then we called `program.translateText` and the others. But now we just need to have the index.js be evaluated.